main.tf

```diff
@@ -27,10 +27,21 @@ locals {
 
 data aws_caller_identity "current" {}
 
+resource "aws_s3_bucket" "postman_bucket_logs" {
+  count = local.using_local_files ? 1 : 0
+
+  bucket = "${var.app_name}-postman-tests-${data.aws_caller_identity.current.account_id}-logs"
+  acl    = "log-delivery-write"
```

Comment:
it doesn't need any other access controls? like an IAM policy/task?

Reply:
I don't think so. This is what I copied from the TF docs. I think the ACL takes care of that.

Reply:
OK cool. Just making sure.